src/assets/20-tech-terms/20-beginner-tech-terms.md

```diff
@@ -0,0 +1,157 @@
+
+`Git`, `Subversion (SVN)`, and `Mercurial` are some of the popular `VCS` tools. For instance, `Git` and platforms like `Github` or `GitLab` provide an opportunity to clone repositories, create branches and merge changes which helps to coordinate to work optimally in terms of teamwork as well as project management.
+Want to geek out even more? Have fun exploring, [Git vs Mercurial vs SVN (Subversion)](https://stackshare.io/stackups/git-vs-mercurial-vs-svn)!
+
+### 6. Porting Commits
+Porting commits is where you move or "port" the changes from one Git branch to another. Usually, this is required if you have any commits in one branch that you want to add to another branch but not by merging the whole branch. Generally, there are two ways of porting commits which are:
+- **Forward-Porting a commit**:
+`Forward-porting` is the practice of moving changes from an older branch to a newer branch. That way any fix or update done will be in the main development branch. This includes finding appropriate commits, cherry-picking them to the branch created for it, resolving any conflicts and running tests thoroughly on these changes to make sure nothing broke during integration. This way, a version of the codebase is made consistent and stable across different implementations.
+
+- **Backporting a commit**:
+`Backporting` is when software has some patches or updates in a recent version and they need to be applied to an older version of the same software. It includes: identifying the patches to be done, adapting those patches to be compatible with older versions of the same product, dealing with any undesired side effects surfaced during the last step and finally deploying the backported patch.
+
+### 7. Bump:
+In general, when developers increase the version number of the software they use the term `Bump`. Most of the time, this version number change also signals changes that have been made to the product. Think of it as updating the version of a book. A higher version number points to a newer version of the software, usually with bug fixes or new features and other improvements. Increasing the version number helps developers/users see that a change has been made and helps them ensure that they are using the most compatible and latest version.
+
+## Testing:
+
+### 8. Quality Assurance (QA):
+ Imagine a team committed to ensuring software is flawless before it reaches clients or end users. That's the world of `Quality Assurance (QA)`! `QA` is not just about testing; it is a practice to ensure that defects are identified and prevented during the development lifecycle. Consider them the gatekeepers of software brilliance. They work in tandem to developers to make sure the software possesses the highest quality standards. From reading requirements to test case designing and early identification of issues, QAs play various roles in a project. Their goal? - delivering a user-friendly and seamless working final product that is close to being perfect.
```

Comment:
not sure if you are referring to QA as a methodology/practice or QA engineers in this paragraph. The first two sentence sound more like methodology/practice, but expressions in the later part sounds like you're referring to the person doing QA. Here is a suggestion to make this clearer. Feel free to use your own words :)

```suggestion
Imagine a team committed to ensuring software is flawless before it reaches clients or end users. That's the world of `Quality Assurance (QA)`! `QA` is not just about testing; it is a practice to ensure that defects are identified and prevented during the development lifecycle. Consider QA engineers the gatekeepers of software brilliance. They work in tandem to developers to make sure the software possesses the highest quality standards. From reading requirements to test case designing and early identification of issues, QAs play various roles in a project. Their goal? - delivering a user-friendly and seamless working final product that is close to being perfect.
```

Reply:
Yeah. I referred to the QAs